### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by sanathkumar26700**
Created 3 variants of card components,
     1. Vertical card components
     2. Horizontal card components
     3. Disabled card components

live preview :  https://deploy-preview-4--viewi.netlify.app/components/card%20components/card-components

### CODE REVIEW

File: Components/card components/card-components.css [hunk 1]
```diff
@@ -1,25 +1,23 @@
 .card {
     display: flex;
     flex-direction: column;
-    border-radius: 0.5rem;
-    flex-basis: (min-content, auto);
+    border-radius: 1rem;
     box-shadow: 1px 1px 2px 1px rgba(0, 0, 0, 0.212);
```
Comment (msdeshmukh009) [inline Components/card components/card-components.css]: You can use variables for box-shadow as well

File: Components/card components/card-components.css [hunk 2]
```diff
@@ -37,36 +35,36 @@
...
 
 /*-----------------Buttons--------------------*/
 
-.btn {
+.card__action--btn {
     background: none;
     background-color: white;
     border: none;
     border-radius: 2px;
     color: #6200ee;
```
Comment (msdeshmukh009) [inline Components/card components/card-components.css]: color variable can be used here

**Review by msdeshmukh009 — COMMENTED** [review]
Maybe make a text-only card as well.

File: Components/card components/card-components.css [hunk 3]
```diff
@@ -37,36 +35,36 @@
...
     cursor: pointer;
     padding: 0.5rem 0.9rem;
-    transition: background-color 2s ease-in;
 }
 
-.btn:hover,
-.card__main:hover,
+.card__action--btn:hover,
 #icon:hover {
     background-color: #d6d6d6;
```
Comment (mimansha-swarup) [inline Components/card components/card-components.css]: use variables

File: Components/card components/card-components.css [hunk 4]
```diff
@@ -199,25 +197,26 @@
...
 }
 
 .card__btn>* {
-    background-color: orangered;
+    background-color: var(--primary-color);
     border: none;
-    padding: 0 0.75rem;
+    padding: 0.5rem 1rem;
     border-radius: 0.5rem;
     box-shadow: 1px 1px 2px 1px rgba(0, 0, 0, 0.212);
```
Comment (mimansha-swarup) [inline Components/card components/card-components.css]: use var for box-shadow too

**Review by mimansha-swarup — COMMENTED** [review]
few suggestions

File: Components/card components/card-components.css (hunk 4 above)
Comment (sanathkumar26700) [reply Components/card components/card-components.css]: Done

File: Components/card components/card-components.css (hunk 2 above)
Comment (sanathkumar26700) [reply Components/card components/card-components.css]: Done

File: Components/card components/card-components.css (hunk 3 above)
Comment (sanathkumar26700) [reply Components/card components/card-components.css]: done

File: Components/card components/card-components.css (hunk 1 above)
Comment (sanathkumar26700) [reply Components/card components/card-components.css]: done

Comment (sanathkumar26700) [reply Components/card components/card-components.css]: > Maybe make a text-only card as well.

Done

**Review by msdeshmukh009 — APPROVED** [review]

**Pull request merged**